bin/fin

```diff
@@ -3275,12 +3275,12 @@ project_create ()
 		2)
 			target_cms="Drupal 9 (BLT Version)"
 			target_host_name_search_string="drupal9.docksal"
-			target_repo="$URL_REPO_DRUPAL9COMPOSER"
+			target_repo="$URL_REPO_DRUPAL9BLT"
```

Comment:
Thank you @shelane for this. I'm using fin 1.99 and just created a #2 project and didn't get BLT installed... I was like "WUT!!" .. then looked at the source code and was about to submit a patch doing what you just did above!  #beat-me-2-it  .. Glad to see it's fixed in the next version!